File: submissions/sm_112_soumik/week_03/day_4/session_1/check_subset.js
function checkSub(set1,subset){
    var sub=[]
    var count = 0
    for(i =0; i< set1.length;i++){
        for(j =0;j<subset.length;j++)
        if(subset.includes (set1[j]))
            count += 1       
    }
    if(count == 0){
        console.log("Not Subset")
    }
    else
        console.log(" Subset")
}
checkSub([10,12,15,18,0,5],[19,0,5])

File: submissions/sm_105_ashish/week_13/day_4/session_2/check_subset.py
set1 = {2,3,7}
set2 = {1,2,3,4,7}

count = 0
for i in set1:
    for j in set2:
        if i == j:
            count += 1
if count == len(set1):
    print("set1 is subset of set2")
else:
    print("set1 is not subset of set2")           